src/pe/mod.rs

```diff
@@ -268,7 +266,11 @@ impl<'a> PE<'a> {
                         bytes,
                         certificate_table.virtual_address,
                         certificate_table.size,
-                    )?;
+                    )
+                    .unwrap_or_else(|err| {
+                        warn!("Cannot parse CertificateTable: {:?}", err);
+                        Default::default()
```

Comment:
I can't remember, what is a default certificate directory in this case? is it going to cause other problems further down the line when parsing, or if a user accesses parts of it, will it panic? Does it have offsets into other parts of the PE file that are no longer valid, etc.?

Reply:
default is empty table (no certificates), so no wrong offsets there